File: Python/Project Euler #2.py
f = [1, 1]
sum = 0
while f[1] < 4000000:
    x = f[1] + f[0]
    f[0] = f[1]
    f[1] = x
    if f[1] % 2 == 0:
        sum = sum + f[1]
print(sum)

File: Javascript/Project Euler #2.js
var f1 = 1;
var f2 = 1;
var sum = 0;
while (f2 < 4000000)
{
  let x = f1 + f2;
  f1 = f2;
  f2 = x;
  if (f2 % 2 == 0)
    sum = sum + f2;
}
console.log(sum)
